### ISSUE TITLE
[Blood] Support more gameplay quirks from older versions of the game

### ISSUE THREAD

**Issue by sinisterseed** [coelckers/Raze#497]
If the required amount of work wouldn't be unrealistic, I'd personally like to have the different properties of the weapons and items from older versions of Blood supported for us too.

Based on the discussions from the isRunning commit (copy pasted below), it'd be those. Seeing how upstream has or had those implemented, the required work shouldn't be unnecessarily big to get going.

> Well, as far as I know in the original 1.00 release the lifeleech did what the name suggested. It damaged enemies and gave you life. If you ran out of souls then it damaged you to fire, like it does now.
> This was patched/broken in future versions. Personally I liked this funtionality as it gave the weapon more of a purpose. While it does great fire damage (and is OK in bloodbath) it is rather pointless in single player I find.
> Apparently it was changed in 1.1:
> "In version 1.0 of Blood, the Life Leech was able to do a Vampiric attack which would heal Caleb up to 100 health points. This mechanic was changed in version 1.1 onwards."
> NBlood allows (or used to) you to select the version 1.00 weapon properties. This changed the voodo doll to use the original logic, the lifeleech to steal life and the akimbo napalm to dual fire, rather than the bomblets it does from the Plasma pak.
> 
> As for the fanatics, you are right. Apparently:
> 1.00 Everyone throws dynamite
> 1.02 Noone throws dynamite
> 1.10 Cultists throw dynamite
> https://twitter.com/SlappyCromwell/status/1071572374460473344

**Comment by Tiddalick** [comment]
Yes it does exist in NBlood already, but it is all the 1.0x version settings or none.
The key term that links it all together is [gWeaponsV10x](https://github.com/nukeykt/NBlood/search?q=gWeaponsV10x)

I think it would be better to allow these individual settings to be changed, perhaps through the CVARs system that was used for the IsrunningAI stuff.

I'm not a programmer so afraid I cannot really do more to assist with the idea.

**Comment by Gaerzi** [comment]
Something like the compatibility setting options in GZDoom: individual setting for each quirks, but with 1.00, 1.02, and 1.10 presets that set all relevant settings at once. So you can mix and match if you want.

**Comment by mjr4077au** [comment]
This has been added in 17b2154e9aba0ca517f07597507949b57253cb58, just leaving the commit open as the langauge spreadsheet will need a definition (if we're going to expose this to the menu).

100% agreed on the bitfield compatibility settings like GZDoom, we just haven't got there yet.

**Comment by sinisterseed** [comment]
This absolutely needs menu entries, so let's make a Compatibility submenu for Blood.

If we go with bitfields, the sheet can wait for a while until we get the toggles set up. I'm way more in favor of customizability, so GZDoom-esque settings over all or nothing is my preference for this.

**Comment by Dzierzan** [comment]
When it comes to Weapons 1.0 balance, Napalm Launcher didn't do any fire damage so it has to be a thing too.
